src/current/v24.3/data-resilience.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Data Resilience
+summary: An overview of how CockroachDB deployments can achieve resilience with built-in features and tooling.
```

Comment:
i think this page should have a TOC to make navigating easier

Reply:
Oh yes, thanks for pointing that out, my original plan for the page wasn't going to have sections, but then it changed!